DocsGeneration/MarkDowns/Parameters.cs

```diff
@@ -5,24 +5,21 @@
 
 namespace DocsGeneration.MarkDowns {
   public class Parameters {
-    public static readonly string ParametersOverview =
-      "# Parameters\n" +
-      "\n" +
-      ":::danger" +
-      "\nGSA-Grasshopper plugin is pre-release and under active development, including further testing to be undertaken. It is provided \\\"as-is\\\" and you bear the risk of using it. Future versions may contain breaking changes. Any files, results, or other types of output information created using the plugin should not be relied upon without thorough and independent checking.\n:::\n" +
-      "\n" +
-      "The GSA plugin introduces a new set of custom Grasshopper parameters. Parameters are what is passed from one component's output to another component's input.\n" +
-      "![Parameters](https://developer.rhino3d.com/api/grasshopper/media/ParameterKinds.png)\n" +
-      "\n" +
-      "## Custom GSA Parameters" +
-      "\n\n";
-
-
     public static void CreateOverview(Dictionary<string, List<Parameter>> parameters) {
       string filePath = @"Output\gsagh-parameters.md";
       Console.WriteLine($"Writing {filePath}");
 
-      string text = ParametersOverview;
+      string text = "# Parameters\n\n";
+      if (GsaGH.GsaGhInfo.isBeta) {
+        text += StringHelper.AddBetaWarning();
```

Comment:
avoid using + to join strings. It's because every single time when you use it, old string is deleted and new one is created. It's not very good for performance. example move + "\n" into next concatenation

Reply:
use StringBuilder when you have a lot of concatenations

Reply:
This PR is not about refactoring the docs generator to use stringbuilder though

Reply:
for the future :p

Reply:
Can you resolve this conversation then?